Source/LoadByOS/LinuxConfigApp/LinuxInterop.cs

```diff
@@ -1,14 +1,73 @@
 ﻿using LibationFileManager;
+using System.Diagnostics;
 
 namespace LinuxConfigApp
 {
     internal class LinuxInterop : IInteropFunctions
     {
-        public LinuxInterop() { }
+		//Different terminal apps possibly installed on a linux system
```

Comment:
Yay! So validating to see 'just in case' infrastructure get proper use. When I added these interop files last year I didn't know if the mac or linux ones would actually get used.

Reply:
I felt the same way about my addition of `.releaseindex.json` lol